### ISSUE TITLE
My game is not recognized / does not provide an ID (Leathal Company Vr Mod)

### ISSUE THREAD

**Issue by Marwasp** [BOLL7708/OpenVR2Key#105]

**Comment by BOLL7708** [comment]
Does the application work with other native VR games? It is required for SteamVR to recognize an application as a scene application and provide the app ID for it to get separate bindings. 

Are you running this from Steam, externally to steam, or what are you actually doing? This is why actually providing a description and details in an error report helps people help you, keep that in mind.

**Comment by Marwasp** [comment]
Yes the application works on native Vr games. I use the Thunderstore Mod Manager to load the mods for Leathal company and the game starts useing Steam. Leathal company is not a native Vr game. Anything else you would like to know?
